build/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM python:3.8.3
+ARG version=latest
```

Comment:
Maybe this is a detail but why is the default in the Dockerfile `latest`?

Reply:
It's arbitrary, but nice to have a default so if we do a `docker build` with no `--build-arg` we still get something useful. Note, we also duplicate that default value in the workflow `inputs`, since the [workflow-trigger](https://github.com/Breakthrough-Energy/actions/blob/main/workflow-trigger/action.yml) action doesn't yet support that.